XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -192,7 +184,7 @@ public void ShowLocalValidationMessages()
 
         public void HideLocalValidationMessages()
         {
-            if (InvalidParamToolTip.Tag is Control ctrl && ctrl != null)
+            if (InvalidParamToolTip.Tag is Control ctrl)

```

Comment:
The null check at the end is not necessary because the statement `is Control ctrl` checks implicitly.